src/ci/docker/dist-various-2/build-x86_64-fortanix-unknown-sgx-toolchain.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution and at
+# http://rust-lang.org/COPYRIGHT.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+set -eu
+source shared.sh
+
+if [ -z $1 ]; then
+    echo "Usage: ${0} <libunwind_commit_id>"
+    exit -1
+fi
+
+set -x
+libunwind_commit_id=$1
+target="x86_64_fortanix_unknown_sgx"
+
+install_prereq()
+{
+    apt-get update
+    apt-get install -y --no-install-recommends \
+            build-essential \
+            ca-certificates \
+            cmake \
+            git
+}
+
+build_unwind()
+{
+    dir_name="${target}_temp"
+    mkdir -p ${dir_name}
+    cd ${dir_name}
+
+    # Clone LLVM and libunwind
+    # libunwind cmake build needs cmake configuration from LLVM.
+    git clone --depth 1 https://github.com/llvm-mirror/llvm.git
```

Comment:
Hm, could we perhaps reuse the existing llvm submodule (src/llvm)? If not, a comment here why not would be good.

Cloning llvm twice wouldn't really be ideal (it takes a while to do so on CI)

Reply:
@Mark-Simulacrum 
We found a way to use existing LLVM. Please let us know what you think about the current approach.